src/main/java/bit/javaoop/Manager.java

```diff
@@ -0,0 +1,35 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Manager extends Employee {
+
+    static List<String> managers = new ArrayList<>();
+    Manager(String name, double salary){
+        super(name,salary);
+        managers.add(name);
+    }
+
+   boolean isHire(String name, double salary)
+    {
+        int spots = 100;
+        boolean question;
+        if (spots > 0) {
+            question = true;
+            spots -= 1;
+        } else {
+            question = false;
+        }
+
+        return question;
+    }
+
+    void addEmployee(String name, double salary) {
+        List lista = new ArrayList();
+        if (isHire(name, salary) == true) {
+            lista.add(name);
```

Comment:
No to jest ciekawa metodka 😄 

Tworzysz lokalną listę, do której dodajesz pracownika, po czym lista znika bezpowrotnie, gdy wyjdziemy z metody :P

Po zastosowaniu mojej powyższej uwagi o dodaniu listy pracowników zaimplementowanie tej metody byłoby oczywiste. A tak rozumiem Cię, że nie bardzo wiedziałaś jak to zrobić. Bo zamysł jest dobry. Ogólnie wyglądałoby to tak:
![image](https://user-images.githubusercontent.com/26125971/38339648-a5e04fc0-386f-11e8-8afc-1e223b9dab20.png)
Taka implementacja wymagałaby zamiany dwóch parametrów przyjmowanych przez `isHire` na jeden o typie `Employee` (oczywiście nie trzeba tego robić mógłbym przepisać to tak jak Ty napisałaś  z imieniem i pensją, ale jak mówiłem powyżej, bardziej obiektowo jest z użyciem wyższej abstrakcji jaką jest obiekt `Employee` :))